thirdparty/build-thirdparty.sh

```diff
@@ -851,7 +851,7 @@ build_opentelemetry() {
         -DCMAKE_CXX_STANDARD="17" \
         -DCMAKE_INSTALL_PREFIX=${TP_INSTALL_DIR} \
         -DBUILD_TESTING=OFF -DWITH_EXAMPLES=OFF \
-        -DWITH_STL=ON -DWITH_JAEGER=ON
+        -DWITH_STL=OFF -DWITH_JAEGER=ON
```

Comment:
This looks like a bug! problem description reference: https://github.com/open-telemetry/opentelemetry-cpp/issues/1200

Reply:
You should open another PR to modify thirdparty build